src/handlers/opensea-api.ts

```diff
@@ -83,14 +88,16 @@ export const apiGetAccountUniqueToken = async (
     const urlV2 = `https://${NFT_API_URL}/api/v2/beta/asset/${contractAddress}/${tokenId}${
       forceUpdate ? '?force_update=true' : ''
     }`;
-    const { data } = await rainbowFetch(isPolygon ? urlV2 : url, {
-      headers: {
-        'Accept': 'application/json',
-        'X-Api-Key': NFT_API_KEY,
-      },
-      method: 'get',
-      timeout: 10000, // 10 secs
-    });
+    const dataTask = async () =>
+      await rainbowFetch(isPolygon ? urlV2 : url, {
+        headers: {
+          'Accept': 'application/json',
+          'X-Api-Key': NFT_API_KEY,
+        },
+        method: 'get',
+        timeout: 10000, // 10 secs
+      });
+    const { data } = await queue.add(dataTask);
```

Comment:
can we just inline the async function instead of defining `dataTask`?

Reply:
🫡